elastic/kibana
# [Synthetics UI] Add/Edit monitor - add loading state to add/edit button

## Summary

Adds loading state to Synthetics add and Synthetics edit button

<img width="1389" alt="Screen Shot 2022-08-26 at 11 33 42 AM" src="https://user-images.githubusercontent.com/11356435/186941138-c794eddc-8e0c-4375-904f-56e22401ff3f.png">

### Testing
1. Navigate to Synthetics UI
2. Add a monitor. Confirm loading state for the button
3. Edit the monitor. Confirm loading state for the button
